### ISSUE TITLE
Tire TPU infill

### ISSUE THREAD

**Issue by tom-f-oconnell** [XRobots/ReallyUsefulRobot#7]
What infill did you print your TPU tires with?

Googling around it seems like the 85A hardness Ninjaflex is the most common (and perhaps also the only one called *just* "Ninjaflex", but I'm not sure). Do you know whether this is also the hardness of the Ninjaflex you used?

Thanks again.

**Comment by XRobots** [comment]
Yep that's the one - just the standard stuff. It was 4 perimeters and around 40% infill - whatever made each tyre come out of one roll of filament basically. Previously I've used up to 80% infill for tyres which is practically solid.
